site/gatsby-site/playwright/config.ts

```diff
@@ -11,6 +11,7 @@ const config: ConfigType = {
     E2E_ADMIN_USERNAME: process.env.E2E_ADMIN_USERNAME!,
     IS_EMPTY_ENVIRONMENT: process.env.IS_EMPTY_ENVIRONMENT ?? '',
     AVAILABLE_LANGUAGES: process.env.GATSBY_AVAILABLE_LANGUAGES ?? '',
+    MONGODB_CONNECTION_STRING: process.env.MONGODB_CONNECTION_STRING ?? '',
```

Comment:
please add this as an optional type to the `ConfigType` interface above, like `AVAILABLE_LANGUAGES`. (and see with below comment -- this might be a good change to keep if its documenting an env variable that's being used)